app/main.py

```diff
@@ -1,8 +1,47 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> float:
+        price = (car.comfort_class * (self.clean_power - car.clean_mark)
+                 * self.average_rating / self.distance_from_city_center)
+
+        return round(price, 1)
+
+    def wash_single_car(self, cars: list) -> float:
+        total_income = 0.0
+
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income = self.calculate_washing_price(car)
+                total_income += income
+                car.clean_mark = self.clean_power
+        return total_income
+
+    def serve_cars(self, cars: list) -> float:
```

Comment:
The `serve_cars` method should ensure that only cars with `clean_mark` less than `clean_power` are washed. This logic is correctly implemented in `wash_single_car`, but ensure that `serve_cars` is intended to call this method to process a list of cars .